### DISCUSSION TITLE
I am getting a TypeError error

### DISCUSSION THREAD

**Discussion by serkanbayburtlu** [vercel/vercel#4648] [Help]
Hi,

I can build and run the project seamlessly on localhost. Also I'm not getting any errors.

But when I want to deploy the project, I get an error. 

Is there anything I missed?

Thanks you,

VS Terminal
![local](https://user-images.githubusercontent.com/40633611/84565925-cfa12200-ad64-11ea-8c30-87664abbff2d.png)

Vercel Build Logs
![vercel](https://user-images.githubusercontent.com/40633611/84565926-d2037c00-ad64-11ea-960d-5a48cf29edb6.png)

**Comment by paulogdm** [comment]
Can you provide some code so we can assist you further?

**Comment by markozxuu** [comment]
@serkanbayburtlu make sure you are complying with all typescript type rules. On some occasions `next dev` does not report somes kind of problems that you may have when making a production deploy
